joblib/test/test_memory.py

```diff
@@ -141,6 +141,23 @@ def g(l=None, m=1):
     assert g(l=30, m=2) == 30
 
 
+def test_in_store(tmpdir):
+    " Test in_store function."
+
+    def g(a, b, c, d, e=None, f=1):
+        return (a, b, c, d, e, f)
+
+    memory = Memory(location=tmpdir.strpath, verbose=0)
+    g = memory.cache(g)
+    args = (1, 2, 3, 4)
+    kwargs = {'e': 5, 'f': 6}
+    g(*args, **kwargs)
+    assert(g.in_store(*args, **kwargs))
```

Comment:
no need for parenthesis with `assert`. Use the following instead (and be consistent with other uses of `assert`):
```python
assert g.in_store(*args, **kwargs)
```